### PR TITLE
Deprecate skipSauceLabs for integration tests, since they only ever run on sauce labs

### LINKED ISSUE #10970: Integration tests that do `skipSauceLabs()` used to run on Travis but no longer run at all
integration tests used to run as part of `gulp test` which means the unit_test shard used to run them on Travis. Some of them had `skipSauceLabs()` which means they used to only run on Travis with latest Chrome.

With the new `--unit` change and from what I see in the logs, now the `unit_test` shard only runs unit tests and the `integration_test` shard only runs with `--saucelabs` which means all the integration tests with `skipSauceLabs()` no longer run at all.

### PR DESCRIPTION
**Pull request by rsimha-amp**
Follow up to #10825 and #10800 
Skipped tests being tracked by #10974
Fixes #10970

### CODE REVIEW

File: test/integration/test-video-manager.js [hunk 1]
```diff
@@ -114,8 +114,8 @@ describe.configure().ifChrome().skipOldChrome().run('VideoManager', function() {
 
     });
 
-    it.configure().ifChrome().skipOldChrome().run(`autoplay - autoplay not
-        supported should behave like manual play`, () => {
+    it.skip(`autoplay - autoplay not supported should behave
```
Comment (choumx) [inline test/integration/test-video-manager.js]: Please add `TODO(aghassemi)` for these since Ali gracefully volunteered to look.

Comment (aghassemi) [inline test/integration/test-video-manager.js]: created https://github.com/ampproject/amphtml/issues/10974 do `TODO(aghassemi, #10974)`

Comment (rsimha-amp) [inline test/integration/test-video-manager.js]: Done.

**Comment by rsimha-amp** [comment]
Added `ifNewChrome`.

**Comment by rsimha-amp** [comment]
Integration tests are green (https://travis-ci.org/ampproject/amphtml/jobs/265753154), and we're now running about 100 tests that we weren't running earlier.

Good catch, @aghassemi!

**Pull request merged**